### DISCUSSION TITLE
How can I access the "/assets" folder from nuxt.config.js?

### DISCUSSION THREAD

**Discussion by marquizzo** [nuxt/nuxt#19345] [Questions (Nuxt 2)]
I'm using Nuxt 2.15. I'm trying to preload a font with a `<link>` tag, so I add the following to my `app.html` file:

```
<head {{ HEAD_ATTRS }}>
  {{ HEAD }}
  <link rel="preload" href="/assets/fonts/SoDoSans.woff" as="font" type="font/woff">
</head>
```
However, this gives me a 404 error because the `/assets/` folder doesn't exist. So I tried `nuxt.config` instead:
```lang-js
link: [
  { rel: 'preload', type: 'font/woff', href: '/assets/fonts/SoDoSans.woff', as: "font" },
]
```
But once again, that assets folder doesn't exist. I can't hard-code the URL either, because that file gets converted to `SoDoSans.2ae2253.woff` on production. 

How can I access files in the `/assets` folder from nuxt.config or app.html?

**Comment by Baroshem** [comment]
Hey,

How about using this?

https://nuxtjs.org/docs/directory-structure/assets#fonts

**Comment by Baroshem** [comment]
Or actually this in html

```
<img :src="require(`~/assets/img/${image}.jpg`)" />
```

But instead of this image, use the corresponding tag and path

**Reply by marquizzo** [reply]
Thank you, but neither of these suggestions answer the original question.
1. I already have fonts in CSS via `@font-face`. What I'm trying to do is getting the location of those font files so I can add them to a `<link rel="preload">` tag in `app.html` for early preloading.
2. `require()` does not work on `app.html`. When I try to bind VUE variables in app.html, the output is a literal `<img :src="require('~/assets/image.jpg')">` which tells me that `app.html` doesn't get processed by Nuxt in the same way VUE files do.
